### DISCUSSION TITLE
Different folders or tags or any other feature to group sites

### DISCUSSION THREAD

**Discussion by waseeen** [0xJacky/nginx-ui#613] [Ideas]
I have been using Nginx for a year now and have created many configurations for different projects, each project has many configurations. Since I started using NginxUI, I have been wondering why there is no option to group/sort the list of sites. This would be very useful for those who keep multiple projects. The way I'm using now to achieve the same behavior is to add some tag before the configuration name so that they are sorted alphabetically
![image](https://github.com/user-attachments/assets/d69a6b03-5e73-46dc-845e-b51f6248cf5a)

**Comment by 0xJacky** [comment]
Hi @waseeen, this feature will be implemented in next beta version.

**Reply by waseeen** [reply]
Amazing! Looking forward for it
